test/commands/generate/fromTemplate.test.ts

```diff
@@ -95,7 +95,7 @@ describe('template', () => {
       .it('should print debug logs', (ctx, done) => {
         expect(ctx.stdout).toContain(
           `Template sources taken from ${path.resolve(
-            './test/minimaltemplate'
+            './node_modules/@asyncapi/minimaltemplate'
```

Comment:
It probably causes the failing tests. Why did you change path?

Reply:
Just because it was failing. But it got passed after that change 🤔 Let me take a look at it 👀

Reply:
After reverting that change now the test is passing. Seems to be failing just at my local but don't understand why. Should I make some change on my local repo folder's structure?